app/pods/components/agenda/agenda-detail/sidebar-item/template.hbs

```diff
@@ -38,48 +38,46 @@
 
           <div class="auk-u-flex auk-u-flex--justify-between">
             <div class="auk-u-flex auk-u-flex--align-end vlc-agenda-detail-sidebar-item__icon">
-              {{#if this.renderDetails}}
```

Comment:
Waarom is deze eruit ? dit was een fix dat de detailview ineens al deze data gaat ophalen voor alle agendapunten, ook al zijn die niet in view